app/imports/components/Users/AdminPanel/UsersManagement/UsersList/usersList.jsx

```diff
@@ -35,11 +57,15 @@ export class UsersList extends React.Component {
     render() {
         return (
             <div>
-                <DropDownMenu maxHeight={300} value={this.state.value} onChange={this.handleChange}>
+                <this.SelectableList defaultValue={3}>
```

Comment:
`this.SelectableList` looks very bad, could you explain why you use it in this way?